scripts/gendocs/generate

```diff
@@ -174,9 +177,9 @@ rm -r pkg/apis/certmanager/v1alpha3
 rm -r pkg/apis/certmanager/v1beta1
 gendocs "v1.6-docs"
 
-# when adding a new version here run
-# ./scripts/gendocs/generate 1 to force generation of CLI docs
-# and add them to the git index
+# when adding a new version here use
+# genversionwithcli to force generation of CLI docs
+# and add them to the git index, then switch back to genversion
```

Comment:
I don't understand this comment. 
Does it mean that I should add another `genversionwithcli` line somewhere in the scipt.
Please add a little more detail about how this script should be modified in future.

Reply:
I have re-formulated the comment and included an example for the next change in this section